.github/workflows/build.yml

```diff
@@ -10,13 +10,29 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - id: build
+      - id: build-paper-qa-pymupdf
```

Comment:
In the end, will the user do `pip install paper-qa[pymupdf]` or `pip install paper-qa-pymupdf`?